collector/src/main/java/com/bitmovin/analytics/error/ExceptionMapper.kt

```diff
@@ -2,6 +2,6 @@ package com.bitmovin.analytics.error
 
 import com.bitmovin.analytics.data.ErrorCode
 
-interface ExceptionMapper<in T : Throwable> {
+interface ExceptionMapper<in T> {
```

Comment:
Why did you remove this here? You should probably add the ExoPlayer subclass of ExeptionMapper have the `T : Throawable` since you removed it to be able to extend the bitmovin player here

Reply:
Removed to be able to pass `ExoplayerPlabackException` and `ErrorEvent` as type. The `ExoPlayerExceptionMapper` already has `Throwable` as type.